packages/grid/_modules_/grid/context/GridRootPropsContext.ts

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { GridComponentProps } from '../GridComponentProps';
+
+const GridRootPropsContext = React.createContext<GridComponentProps | undefined>(undefined);
```

Comment:
This context stores the props of the `GridComponent`, so maybe `GridComponentPropsContext`?

Reply:
I proposed "root" to signal that it contains he top level props.

Reply:
Well, in this context it makes sense. I suggested a different name because the first time I saw I thought: are these the props for the `GridRoot`?